temoa/cli.py

```
@@ -582,11 +584,20 @@ def _copy_tutorial_resources(target_config: Path, target_database: Path) -> None
 
-        with sqlite3.connect(target_database) as conn:
+        with contextlib.closing(sqlite3.connect(target_database)) as conn:
+            conn.executescript(schema_content)
+            conn.execute('PRAGMA foreign_keys = OFF;')
             conn.executescript(sql_content)
+            conn.execute('PRAGMA foreign_keys = ON;')
+            fk_violations = conn.execute('PRAGMA foreign_key_check;').fetchall()
+            if fk_violations:
+                raise sqlite3.IntegrityError(
+                    f'Foreign key check failed after tutorial data load: {fk_violations[:5]}'
+                )
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)